components/molecules/ProfileLanguageChart/profile-language-chart.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useEffect, useState } from "react";
+
+import { AllSimpleColors, LanguageObject } from "../CardHorizontalBarChart/card-horizontal-bar-chart";
+
+import colors from "../../../lib/utils/color.json";
+
+const languageToColor: AllSimpleColors = colors as AllSimpleColors;
+const NOTSUPPORTED = "#64748B";
+interface ProfileLanguageChartProps {
+  languageList: LanguageObject[];
+}
+const ProfileLanguageChart = ({ languageList }: ProfileLanguageChartProps) => {
+  const sortedLangArray = languageList.sort((a, b) => b.percentageUsed - a.percentageUsed);
+  const [percentage, setPercentage] = useState<any>(0);
+
+  useEffect(() => {
+    const totalSumOfFirstFivePercentage = sortedLangArray
+      .slice(0, 5)
+      .map((lang) => lang.percentageUsed)
+      .reduce((prev: number, next: number) => prev + next); // need some help fixing this type error, used any to bypass 🙏
```

Comment:
Is this note still needed?

Reply:
Yes, but is the note about the error still needed

Reply:
Ooh... sorry, didn’t notice that still exist 😁